app-shells/mksh/mksh-54.recipe

```diff
@@ -39,7 +39,7 @@ BUILD_REQUIRES="
 BUILD_PREREQUIRES="
 	cmd:awk
 	cmd:cmp
-	#cmd:ed
+	cmd:ed
```

Comment:
You are right, there is no real reason for that.
Actually, I guess my intention was to save @dash102 the hassle of having to find another recipe to fix, if she would simplify the mksh-54.recipe by removing a potentially unneeded dependency.
So I agree with you: there is no need to touch mksh in this PR.